.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,53 @@
-# Contributing
```

Comment:
We'll lose instructions for generation if we overwrite this with the one from synthtool.

Perhaps exclude it from synth.py:

```python
s.copy(templates, excludes=[".github/CONTRIBUTING.md"])
```